lib/oauthUtil.js

```diff
@@ -43,7 +43,19 @@ function loadPopup(src, options) {
   var title = options.popupTitle || 'External Identity Provider User Authentication';
   var appearance = 'toolbar=no, scrollbars=yes, resizable=yes, ' +
     'top=100, left=500, width=600, height=600';
-  return window.open(src, title, appearance);
+  var isIE = document.documentMode && document.documentMode <= 11;
+
+  if (isIE) {
```

Comment:
Curious if it would make sense to put IE detection inside of `lib/util.js`?

If we run into any other scenarios where we need to detect if we support a specific version of IE, it may be useful.